test/logger-usage/src/main/java/org/elasticsearch/test/loggerusage/ESLoggerUsageChecker.java

```diff
@@ -59,6 +57,7 @@
     public static final List<String> LOGGER_METHODS = Arrays.asList("trace", "debug", "info", "warn", "error");
     public static final String IGNORE_CHECKS_ANNOTATION = "org.elasticsearch.common.SuppressLoggerChecks";
 
+    @SuppressForbidden(reason = "command line tool")
```

Comment:
The logger usage checker is a standalone tool. Its main sources have no dependencies besides ASM framework (its test sources reuse our test framework however). Just one additional "SuppressForbidden" beside the other 5 in the classpath ;-)